tests/metrics/network/network-metrics-iperf3.sh

```diff
@@ -42,10 +42,12 @@ function setup {
 function iperf3_bandwidth {
 	setup
 	bandwidth_result=$(mktemp)
-	$DOCKER_EXE run -d --name=iperf3-server ${image} bash -c "mount -t ramfs -o size=20M ramfs /tmp && iperf3 -p ${port} -s" > /dev/null && \
-	server_address=`$DOCKER_EXE inspect --format "{{.NetworkSettings.IPAddress}}" $($DOCKER_EXE ps -ql)` && \
-	$DOCKER_EXE run -ti --rm --name=iperf3-client ${image} bash -c "mount -t ramfs -o size=20M ramfs /tmp && iperf3 -c ${server_address} -t ${time}" > "$bandwidth_result"
-	total_bandwidth=`cat $bandwidth_result | tail -n 3 | head -1 | awk '{print $(NF-2), $(NF-1)}'`
+	server_command="mount -t ramfs -o size=20M ramfs /tmp && iperf3 -p ${port} -s"
+	$DOCKER_EXE run -d --name=iperf3-server ${image} bash -c "${server_command}" > /dev/null && \
```

Comment:
@GabyCT I'm not sure the && gives you the synchronisation you are hoping for - I don't believe the sub commands 'wait for each other' due to && or are run in parallel - the `bash` manual states I believe they have 'left associativity', and for a command `command1 && command2`, command2 will execute iff command1 succeeds. Thus, && gives you sequence - but so will placing the commands on separate sequential lines. It will catch errors and not run subsequent sub-commands, which is good, but I suspect setting `-e` at the top of the script may actually be a better solution for error handling here.
 I might be missing something here though.
Think I'll ask @jodh-intel if he has any thoughts as well.